pages/index.js

```diff
@@ -84,13 +67,11 @@ export default function Home() {
 
       totalGifCount = account.gifList.length;
       console.log("totalGifCount ", totalGifCount)
-
-
       console.log("got the account", account);
       setGifList(account.gifList);
     } catch (e) {
       console.log(`Error in get gifs: ${e}`);
-      setGifList(null)
+      setGifList([])
```

Comment:
gifList used to render and without defensive checks added to renderer it'll fail, instead setting it to an empty array is more error prone